resources/public/api/conf/1.0/endpoints/charitableGiving_amend.raml

```diff
@@ -0,0 +1,40 @@
+is:
+  - headers.acceptJson
+  - headers.contentTypeJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
+  - errors.formatValue
+  - errors.formatString
+  - errors.ruleGiftAidNonUkAmountWithoutNames
+  - errors.ruleGiftAidNonUkInvestmentsAmountWithoutNames
```

Comment:
The error `RULE_GIFTS_NON_UK_INVESTMENTS_NAMES_WITHOUT_AMOUNT` is crossed out in the spec and the RAML is currently missing `RULE_GIFTS_NON_UK_AMOUNT_WITHOUT_NAMES`